dist/lockup.js

```diff
@@ -156,27 +154,9 @@
                     .height(window.innerHeight)
                     .css('overflow', 'hidden')
                     .scrollTop(this.scrollPosition - getPadding('top') - getPadding('bottom'));
-
```

Comment:
@kpeatt I think you might've had some concerns about this, let's chat when you have a moment